honeybee_vtk/writers.py

```diff
@@ -47,23 +53,24 @@ def write_polydata(
 
 
 def write_points(
-        points: List[List], vectors: List[List], vtk_writer, vtk_extension,
-        target_folder, file_name='Grid points'):
+        points: List[List], vectors: List[List], file_name, target_folder,
+        vtk_writer, vtk_extension):
```

Comment:
Right now it's ok but changing the order of arguments will be a breaking change and will make so many people angry! Using a * between the first two inputs and the rest will give you the freedom to update the arguments as you go without breaking someone's code.

```suggestion
        points: List[List], vectors: List[List], *, file_name, target_folder,
        vtk_writer, vtk_extension):
```